packages/docusaurus-theme-classic/src/validateThemeConfig.ts

```diff
@@ -359,6 +366,15 @@ export const ThemeConfigSchema = Joi.object({
     additionalLanguages: Joi.array()
       .items(Joi.string())
       .default(DEFAULT_CONFIG.prism.additionalLanguages),
+    magicComments: Joi.array()
+      .items(
+        Joi.object({
+          className: Joi.string().required(),
+          line: Joi.string(),
+          block: Joi.object({start: Joi.string(), end: Joi.string()}),
```

Comment:
can we provide a block without a required start/end? TS type seems to say no

Reply:
No, but seems reasonable to